.travis.yml

```diff
@@ -38,6 +39,7 @@ env:
     - PATH="~/.cabal/bin/:$(echo $PATH | sed 's/::/:/')"
     - NVM_DIR="${TRAVIS_BUILD_DIR}"
   matrix:
+    - MAKE_RELEASE=true GIT_EDITOR="sed -i '1 s/^/99.99.99 make release test/'"
```

Comment:
could we move the `GIT_EDITOR` var to the script command instead of in the matrix?